dev/com.ibm.ws.security.wim.adapter.ldap/resources/com/ibm/ws/security/wim/adapter/ldap/resources/LdapUtilMessages.nlsprops

```diff
@@ -127,9 +127,9 @@ EXT_ID_HAS_MULTIPLE_VALUES=CWIML4528E: The search operation could not be complet
 EXT_ID_HAS_MULTIPLE_VALUES.explanation=The LDAP attribute used as the external identifier must be unique and cannot contain multiple values.
 EXT_ID_HAS_MULTIPLE_VALUES.useraction=Ensure that the correct attribute is chosen for the external identifier. If no appropriate attribute exists, the distinguished name can be used as the external identifier.
 
-INVALID_BASE_ENTRY_DEFINITION=CWIMK0002E: The user registry operation could not be completed. The configured base entry definition is not valid: {0}. Configure a valid base entry definition in the server.xml file. For example: <baseEntry name="..." baseDN="...."/>.
-INVALID_BASE_ENTRY_DEFINITION.explanation=The syntax for defining a base entry is <baseEntry name="..." baseDN="...."/>.
-INVALID_BASE_ENTRY_DEFINITION.useraction=Fix the base entry definition in the server.xml file.
+INVALID_BASE_ENTRY_DEFINITION=CWIMK0002E: The user registry operation could not be completed. The base entry definition {0} is not valid.  Correct the base entry definition in the server.xml file.
+INVALID_BASE_ENTRY_DEFINITION.explanation= The specified definition of the base entry is incorrect.
```

Comment:
INVALID_BASE_ENTRY_DEFINITION.explanation= The specified definition of the base entry is invalid.

Reply:
Updated.